providers/certs.rb

```diff
@@ -175,6 +175,13 @@
     end
   end
 
+  # Download azure root CA cert
+  remote_file '/tmp/DigiCertGlobalRootG2.crt' do
```

Comment:
Could you please create another `action` to download and import Azure CA?

In a week from now we won't even remember we are abusing the action called `generate_int_certs`